docs/languages/cpp.md

```diff
@@ -10,13 +10,13 @@ MetaDescription: Find out how to get the best out of Visual Studio Code and C++.
 
 # C/C++ for VS Code (Preview)
 
-C/C++ support for Visual Studio Code is provided today as a preview of our work to enable cross-platform C and C++ development using VS Code on Windows, Linux, and OS X. Our focus in this preview release is code editing and navigation support for C and C++ code everywhere that VS Code runs, as well as debugging on Linux, OS X, and Windows (GDB only with Cygin and MinGW).
+C/C++ support for Visual Studio Code is provided today as a preview of our work to enable cross-platform C and C++ development using VS Code on Windows, Linux, and OS X. Our focus in this preview release is code editing and navigation support for C and C++ code everywhere that VS Code runs, as well as debugging on Linux, OS X, and Windows.

```

Comment:
Just came from a marketing meeting, they are changing the rules and we aren't supposed to use "VS Code" any longer.  Either "Visual Studio Code" or "Code".  Given that we have now added full support for Windows I would rephrase the last sentence to:  Our focus in this preview release is code editing, navigation, and debugging support for C and C++ code everywhere that VS Code runs